functions/src/index.ts

```diff
@@ -60,13 +60,17 @@ exports.api = functions
  */
 exports.scheduledFunctionCrontab = functions
   .region('europe-west1')
+  .runWith({
+    timeoutSeconds: 240,
+  })
   .pubsub.schedule('0 21 * * *')
   .timeZone('Europe/London')
   .onRun(async () => {
     try {
       await callCarAPI();
       console.log('Called car API Successfully');
+      return Promise.resolve();
     } catch (error) {
-      console.log(`Function error ${error}`);
+      return Promise.reject(error);
```

Comment:
Rejecting here means that an error is created in the logs. We can then setup alerting based on the error(s)